segurata/enginee/enginee/spiders/auth.py

```diff
@@ -3,11 +3,14 @@
 import scrapy
 import logging
 from typing import Tuple, ClassVar
+from scrapy import Request
 from scrapy.exceptions import CloseSpider
 from scrapy.spiders import CrawlSpider, Rule
 from scrapy.linkextractors import LinkExtractor
 from enginee.user_config.parse import user_config_parse
 from enginee.user_config.exceptions import *
+from enginee.login.login import *
```

Comment:
I would try not to use the `*` operator because it's less explicit. The only exception I find feasible is the `exceptions` (haha) module, because in the future there can be TONS of them. What do you think?

```suggestion
from enginee.login.login import LoginGeneralError
```